EIPS/eip-4762.md

```diff
@@ -268,6 +254,13 @@ class AccessSubtree(Container):
     elements: BitVector[256]
 ```
 
+## Block-level operations
+
+* Precompile accounts, system contract accounts and slots of a system contract that are accessed during a system call are considered warm.
+* When (and only when) calling a system contract _via a system call_, the code chunks and account should not appear in the witness.
+* The coinbase account is warmed before the block, and the whole account is accessed
+* Withdrawals are cold, but are added to the witness at no cost if they are the target of a transaction.
```

Comment:
we should add the clarification about storage cells here.
If in a trasnaction, storage value is set and then reset to zero, we should insert zero to the tree and not leave it empty because the user paid the gas cost.
Basically if we charge CHUNK_FILL_COST, fill the chunk value.

Reply:
lateral question/observation to the above:

this is just because withdrawals are processed at the end of the block, and there is no actual cost charged to the coinbase for processing them?